src/status_im/ui/screens/group/chat_settings/events.cljs

```diff
@@ -11,8 +11,9 @@
  :show-group-chat-profile
  (fn [{:keys [db] :as cofx} [_ chat-id]]
    (fx/merge cofx
-             {:db (assoc db
-                         :new-chat-name (get-in db [:chats chat-id :name]))}
+             {:db (-> db
+                      (assoc :new-chat-name (get-in db [:chats chat-id :name]))
+                      (assoc :current-chat-id chat-id))}
```

Comment:
could you please elaborate on this change?

Reply:
group profile view uses `:current-chat-id` to populate it's content. Without it, if user had navigated to public or private chat prior to long pressing on group chat in home to go to group chat's profile, the profile view would try to show profile of public/private profile and not group's profile - ending with error case

Reply:
didn't have to worry about it *before* because, the only way to view group profile was from a particular group chat view, when `:current-chat-id` would have been ensured to be that of the particular group

Reply:
i'm afraid it may lead to some issues, because if you close profile `:current-chat-id` will stay there, but there is no current chat maybe we could provide chat-id in screen params for this screen ?

Reply:
or maybe it's fine if `:current-chat-id` always have the latest value idk better to check this

Reply:
I think outside chat stack(i.e. home, browser, wallet, profile) `:current-chat-id` is mostly always something(mostly the last chat visited) - it is set, but never unset - has been that way before this PR

I had the same uncomfortable thought about the way it is

Seems like there could be some refactoring in order about how all that works, but not sure exactly what the structure/semantics ought to be

Reply:
thanks